axes/helpers.py

```diff
@@ -259,7 +259,7 @@ def make_cache_key_list(filter_kwargs_list):
     return cache_keys
 
 
-def get_client_cache_key(
+def get_client_cache_keys(
     request_or_attempt: Union[HttpRequest, AccessBase],
     credentials: Optional[dict] = None,
 ) -> str:
```

Comment:
Since we returning list of keys, type hint should be list[str]

Reply:
Yeah good catch. I was looking at the docstrings, naming and type hints and they could use a check-up to verify correctness. I've become a bit too accustomated to the code base and am not catching them with a fresh eye anymore.